presentation/src/main/java/com/example/user/test/presentation/screen/splash/SplashActivity.kt

```diff
@@ -21,8 +22,12 @@ class SplashActivity : BaseActivity<SplashPresenter, FlowRouter>(), SplashView {
         const val SPLASH_ACTIVITY_REQUEST_CODE = 1
     }
 
+    init {
+        MarvelApplication.appComponent.inject(this)
```

Comment:
You must inject according to `Fragment/Activity` lifecycle. Move this logic to `onCreate`.